src/hooks/transactions/useSignTransactions.tsx

```diff
@@ -123,7 +138,9 @@ export const useSignTransactions = () => {
       const errorMessage =
         (error as Error)?.message ||
         (error as string) ||
-        PROVIDER_NOT_INTIALIZED;
+        PROVIDER_NOT_INITIALIZED;
+      console.error(errorMessage);
```

Comment:
oops

Reply:
the console error is added intentionally to log the error from the catch block